archetype-module-content/src/main/resources/META-INF/maven/archetype-metadata.xml

```diff
@@ -0,0 +1,50 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<archetype-descriptor xsi:schemaLocation="http://maven.apache.org/plugins/maven-archetype-plugin/archetype-descriptor/1.0.0 http://maven.apache.org/xsd/archetype-descriptor-1.0.0.xsd"
+                      name="openmrs-archetype-module"
+                      xmlns="http://maven.apache.org/plugins/maven-archetype-plugin/archetype-descriptor/1.0.0"
+                      xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance">
+    <requiredProperties>
+        <requiredProperty key="groupId">
+            <defaultValue>org.openmrs.module</defaultValue>
+        </requiredProperty>
+        <requiredProperty key="artifactId" />
+        <requiredProperty key="version">
+            <defaultValue>1.0.0-SNAPSHOT</defaultValue>
```

Comment:
On this, while there's [an official workaround](https://github.com/apache/maven-archetype/commit/9dc6960ce43c59c0345756de4432a18cc1a8212b), this works:

```
${package.getClass().forName("java.lang.String").getConstructor($package.getClass().forName("java.lang.String")).newInstance("1.0.0-SNAPSHOT")}
```

(Basically, by not having it as a simple string, it actually prompts the user for a version. This may only apply to newer versions of the plugin...

Reply:
i have been at this comment for hours, failed to comprehend what you want here! @ibacher

Reply:
I did that already,  thanks